README.md

```diff
@@ -190,6 +190,7 @@ The following table indicates which providers are supported on each platform. As
 | [cisco_pim_rp_address](#type-cisco_pim_rp_address) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
 | [cisco_pim_grouplist](#type-cisco_pim_grouplist) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
 | [cisco_portchannel_global](#type-cisco_portchannel_global) | ❌* | ✅* | ✅* | ❌* | ❌* | ❌ | * [caveats](#cisco_portchannel_global-caveats) |
+| [cisco_stp_global](#type-cisco_stp_global) | ❌* | ✅* | ✅* | ❌* | ❌* | ❌ | * [caveats](#cisco_stp_global-caveats) |
```

Comment:
Mike, I've put `*` after some of the ❌'s but I only did that because we hadn't completely validated the provider yet and I didn't want to lose this info. 

I was telling Sanjay we need one more icon to use for "supported but not validated" platforms; ie. a yellow caution icon maybe.